src/librustc/hir/mod.rs

```diff
@@ -1812,6 +1812,7 @@ pub struct InlineAsmOutput {
     pub constraint: Symbol,
     pub is_rw: bool,
     pub is_indirect: bool,
+    pub span: Span,
```

Comment:
Looks like there's no need to keep span here additionally, output expressions are already preserved in `hir::ExprKind::InlineAsm` (the first `HirVec<Expr>`), so spans can be taken from there.

Reply:
This addition is needed in https://github.com/rust-lang/rust/pull/54945/files#diff-5b4d01d26caf43976125ba0f877e78c0 (for the nll output) as we have `StatementKind::InlineAsm` available only. `ExprKind::InlineAsm` is accessible for ast-based borrow checker and there it is indeed using the `expr.span` (https://github.com/rust-lang/rust/pull/54945/files#diff-dc986596b3470440badc6fb1a667adca).

Reply:
I see.